### PR TITLE
Increase top/bottom padding

### LINKED ISSUE #2654: Layout errors on the delete dataset modal window
When having affected maps.
The card margin-top / bottom should be 28 and it's way less.
Also some scrollbars appear when they shouldn't
screen shot 2015-03-06 at 16 20 51

### PR DESCRIPTION
**Pull request by viddo**
Fixes #2654

screen shot 2015-03-11 at 13 36 59

compare with previous: screen shot 2015-03-06 at 16 20 51

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2225/))

**Comment by viddo** [comment]
@xavijam review the CSS please, AFAIK we don't have any modals that should need scrolling on x-axis, right?

**Comment by viddo** [comment]
cc @saleiva for reported issue

**Comment by xavijam** [comment]
Have you checked it in the create modal window?

**Comment by viddo** [comment]
retest this please

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2227/))

**Comment by viddo** [comment]
@xavijam Noticed that the scrolling issue also happend on the create dialog now when double-checking, fixed in https://github.com/CartoDB/cartodb/commit/48d66b55ae914fcd6280a065e740f5be4e177105 Apart from that, anything else I should check for?

**Comment by xavijam** [comment]
Nope, AFAIK, there isn't any modal with x scroll, so :+1:

File: app/assets/stylesheets/new_common/dialog.css.scss [hunk 1]
```diff
@@ -23,7 +23,7 @@ body.is-inDialog {
   background: -ms-linear-gradient(top, rgba(white,1) 0%, rgba(white,0.95) 50%, rgba(white,0.9) 100%);
   background: linear-gradient(to bottom, rgba(white,1) 0%, rgba(white,0.95) 50%, rgba(white,0.9) 100%);
   filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#ffffff', endColorstr='#ffffff', GradientType=1 );
-  overflow: scroll;
+  overflow-y: scroll;
```
Comment (xavijam) [inline app/assets/stylesheets/new_common/dialog.css.scss]: BTW, why we set scroll instead of auto?

Comment (viddo) [inline app/assets/stylesheets/new_common/dialog.css.scss]: Yes, you're right, just noticed it too.. I remove it.

Comment (viddo) [inline app/assets/stylesheets/new_common/dialog.css.scss]: Don't know if there is any good reason for it, and it's at least not documented here. :/

Comment (xavijam) [inline app/assets/stylesheets/new_common/dialog.css.scss]: I'd set as auto, x and y, wouldn't you?

Comment (viddo) [inline app/assets/stylesheets/new_common/dialog.css.scss]: Looking in the VCS history I found this: https://github.com/CartoDB/cartodb/commit/3e33603949f522947d4486b9831e1805727db7cc

But from what I can see now it should not be necessary anyway.

**Comment by xavijam** [comment]
:+1:

File: app/assets/stylesheets/new_common/dialog.css.scss (hunk 1 above)
Comment (viddo) [inline app/assets/stylesheets/new_common/dialog.css.scss]: Auto is the default so think removing is fine. :)

Comment (xavijam) [inline app/assets/stylesheets/new_common/dialog.css.scss]: Fair enough.

**Comment by viddo** [comment]
retest this please

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2230/))

**Comment by viddo** [comment]
retest this please

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2236/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2237/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2239/))

**Pull request merged**